charts/nd-common/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: nd-common
 description: A helper chart used by most of our other charts
 type: library
-version: 0.0.10
+version: 0.0.11
```

Comment:
I know I have to bump all the other charts in the repo; didn't want to do that before being sure that this will work.